tutorials/s4sdk-secure-cloudfoundry/s4sdk-secure-cloudfoundry.md

```diff
@@ -90,168 +69,148 @@ This alternative approach requires that you have npm installed on your machine.
 
-```
----
-applications:
-- name: approuter
-  host: approuter-p1942765239trial
-  path: approuter
-  memory: 128M
-  buildpack: nodejs_buildpack
-  env:
-    TENANT_HOST_PATTERN: 'approuter-(.*).cfapps.eu10.hana.ondemand.com'
-    destinations: '[{"name":"app-destination", "url" :"https://firstapp-p1942765239trial.cfapps.eu10.hana.ondemand.com", "forwardAuthToken": true}]'
-  services:
-    - my-xsuaa
-```
-Adapt the **TENANT`_HOST_PATTERN`** in the **`manifest.yml`** file to the host of the Cloud Foundry region you are running on; the above assumes EU (`eu10`).
+    ```yaml
 
-Understanding the AppRouter's **`manifest.yml`** and **`xs-app.json`**:
+    ---
+    applications:
+    - name: approuter
+      routes:
+        - route: approuter-<SUBACCOUNT>.cfapps.<REGION_IDENTIFIER>.hana.ondemand.com
+      path: approuter
+      memory: 128M
+      buildpacks:
+        - nodejs_buildpack
+      env:
+        TENANT_HOST_PATTERN: 'approuter-(.*).cfapps.<REGION_IDENTIFIER>.hana.ondemand.com'
+        destinations: '[{"name":"app-destination", "url" :<APPLICATION_URL>, "forwardAuthToken": true}]'
+      services:
+        - my-xsuaa
+    ```
 
-The **TENANT`_HOST_PATTERN`** is a variable that declares the pattern how multiple tenants in the URL are identified and handled. During runtime, the App Router will match the incoming host against the pattern and tries to extract the regular expression from the pattern. It will use the matched string to delegate authentication to the respective XSUAA tenant (in SAP Cloud Platform the tenant ID corresponds to the subaccount ID. If you desire different URL patterns, you need to change this pattern accordingly.
+    Adapt the file as follows:
 
-Note that the **TENANT`_HOST_PATTERN`** variable is only required in real multi-tenant application, i.e, applications where a physical deployment serves multiple clients from the same deployment. We assume in this blog series that we want to build multi-tenant applications, as we aim towards cloud-native development. However, this variable is not necessary if you have a single-tenant application. To realize this, the **`xs-security`.json** security descriptor may declare tenant-mode: dedicated (see step 5 below).
+      - Just like before provide the values for your account and region
+      - In `destinations` replace `<APPLICATION_URL>` with the actual URL of your previously deployed app.
 
-**Destinations** is a variable that declares the internal routes from the App Router to the underlying backend microservices. As we only have one hello world microservice yet, we define only one destination called app-destination here. This app-destination is referenced by the previously created `xs-app`.json file.
+**Note**: It is mandatory that your app URL conforms the format `<APP_NAME>-<SUBACCOUNT>.cfapps.<REGION_IDENTIFIER>.hana.ondemand.com`. The authentication mechanism will identify you by pulling the user-id (`SUBACCOUNT`) out of the subdomain.
```

Comment:
See above. It is not the user-id that gets pulled out, but the subdomain which is then mapped to the tenant-id.

Reply:
I believe the wording is correct. The subdomain is `approuter-UID`, the pattern `approuter-(.*).cfapps...` handles extracting the UID = tenant id from the subdomain.